src/pages/home/report/ContextMenu/PopoverReportActionContextMenu.js

```diff
@@ -307,6 +311,7 @@ class PopoverReportActionContextMenu extends React.Component {
                         draftMessage={this.state.reportActionDraftMessage}
                         isArchivedRoom={this.state.isArchivedRoom}
                         isChronosReport={this.state.isChronosReport}
+                        childReportID={this.state.childReportID}
```

Comment:
Can you explain why we are passing `childReportID` here? I don't see it used anywhere

Reply:
Same thing here I don't think we need it.